ansible/roles/candlepin/tasks/main.yml

```diff
@@ -9,7 +9,7 @@
   ansible.builtin.include_tasks:
     file: el8/el8_tasks.yml
   when:
-    - ansible_distribution in ["CentOS", "RedHat"]
+    - ansible_distribution in ["CentOS", "AlmaLinux", "RedHat"]
```

Comment:
Do we need to still have here CentOS?

Reply:
no, lets remove it

Reply:
I figured might as well leave it in there because we should still support using CentOS. We just had to switch to AlmaLinux because the CentOS 8 box has the broken repo urls at the moment, but if they ever released a CentOS 8 box with the URLs updated, then people could use that box still.